relayer/relay/src/query/client_consensus_state.rs

```diff
@@ -67,17 +74,15 @@ impl<CS> ConsensusStateResponse<CS> {
     pub fn new(
         client_id: ClientId,
         consensus_state: CS,
-        abci_proof: abci::Proof,
+        abci_proof: Option<abci::Proof>,
```

Comment:
Why isn't the type alias (`CommitmentProof`) used consistently?

Reply:
my bad, changed.